docs/image-verification.md

```diff
@@ -0,0 +1,23 @@
+# Image Verification
+
+To enable image verification, add a stanza like the following to the conainerd config:
+
+```yaml
+[plugins]
+  [plugins."io.containerd.image-verifier.v1.bindir"]
+    enabled = true
+    bin_dir = "/opt/containerd-image-verifier/bin"
+```
+
+Place one or more verifier binaries in `bin_dir` which conform to the following API:
+
+## CLI Arguments
+
+- `-name`: The given reference to the image that may be pulled.
+- `-digest`: The resolved digest of the image that may be pulled.
+
+## Image Pull Judgement
+
+Return an exit code of 0 to allow the image to be pulled and any other exit code to reject the image.
```

Comment:
What is the expected verification the plugin performs based on `name` and `digest`?

Reply:
They can do essentially anything, such as:
1. compare registry against a trusted list of registries
2. block the `latest` tag
3. block pulling images without digest references.
4. check image signatures (this would probably be the most popular use case -- see https://github.com/containerd/containerd/issues/6691 for details)